local/keyword_search.go

```diff
@@ -10,19 +10,7 @@ import (
 	"strings"
 )
 
-// KeywordSearchResult ...
-type KeywordSearchResult struct {
-	XMLName xml.Name `xml:"result"`
-	Meta    struct {
-		TotalCount    int        `json:"total_count" xml:"total_count"`
-		PageableCount int        `json:"pageable_count" xml:"pageable_count"`
-		IsEnd         bool       `json:"is_end" xml:"is_end"`
-		SameName      RegionInfo `json:"same_name" xml:"same_name"`
-	} `json:"meta" xml:"meta"`
-	Documents []Place `json:"documents" xml:"documents"`
-}
-
-// KeywordSearchIterator ...
+// KeywordSearchIterator initializes parameters that used in keyword search.
```

Comment:
`KeywordSearchIterator` is not a function nor a receiver.
`// KeywordSearchIterator is a lazy keyword search iterator.` would be better.